opencti-platform/opencti-graphql/src/http/httpTaxii.js

```diff
@@ -156,9 +160,10 @@ const initTaxiiApi = (app) => {
       const args = rebuildParamsForObject(object_id, req);
       const stix = await restCollectionStix(context, user, id, args);
       const data = R.head(stix.objects);
-      res.set('X-TAXII-Date-Added-First', data?.updated_at);
-      res.set('X-TAXII-Date-Added-Last', data?.updated_at);
-      const versions = data ? [data.updated_at] : [];
+      const updatedAt = getUpdatedAt(data);
+      res.set('X-TAXII-Date-Added-First', updatedAt);
+      res.set('X-TAXII-Date-Added-Last', updatedAt);
```

Comment:
Already problematic before your modification, but this should not be taken for last element?

Reply:
In theory, we'd want to do something like this
```
      const versions = stix.objects.map(o => getUpdatedAt(o));
      res.set('X-TAXII-Date-Added-First', R.head(versions));
      res.set('X-TAXII-Date-Added-Last', R.last(versions));
      res.json({ versions });
```
That is, if OpenCTI supported versioned objects.